src/ui/patch_hub/patch_hub_core.sh

```diff
@@ -159,6 +159,16 @@ function show_registered_mailing_lists()
   message_box='Below, you can see the lore.kernel.org mailing lists that you have registered.'$'\n'
   message_box+='Select a mailing list to see the latest patchsets sent to it.'
 
+  fallback_message='kw could not find any registered mailing list.'$'\n'$'\n'
+  fallback_message+='Try to register mailing lists in the menu "Settings" then ' 
+  fallback_message+='"Register/Unregister Mailing Lists"'
+
+  if [[ "${#registered_mailing_lists[@]}" -eq 0 ]]; then
+      create_message_box 'Error' "${fallback_message}"
+      screen_sequence['SHOW_SCREEN']='dashboard'
+      return "$?"
```

Comment:
There is a good chance that `"$?"` is 0, which means success, and from this code, I don't think you intend to return a success code. Am I correct? If so, you probably need to return 22 here or any other error code.